content/en/docs/tutorials/security/seccomp.md

```diff
@@ -265,6 +265,43 @@ docker exec -it kind-worker bash -c \
 }
 ```
 
+## Create Pod that uses the container runtime default seccomp profile
+
+Most container runtimes provide a sane set of default syscalls that are allowed
+or not. You can adopt these defaults for your workload by setting the seccomp
+type in the security context of a pod or container to `RuntimeDefault`.
+
+{{< note >}}
+If you have the `SeccompDefault` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) enabled, then Pods use the `RuntimeDefault` seccomp profile whenever
```

Comment:
nit:

```suggestion
If you have the `SeccompDefault` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
enabled, then Pods use the `RuntimeDefault` seccomp profile whenever
```

Reply:
Thank @tengqm for taking a look at this. I will update it.

Reply:
I have addressed the comment. PTAL!